### PR TITLE
Updated README and added debug statements for bot1.py (after first group session)

### LINKED ISSUE #21: Bug fixes, housekeeping, environment variables
This is an issue for dependency fixes, syntax fixes, debug statements, environment variables, etc.

### PR DESCRIPTION
**Pull request by taimoorgit**
What this does:

- Add permission info to the readme so we don't have issues running our own bots
- Adds some useful debug statements to bot1.py

### CODE REVIEW

**Comment by OneHandKlap** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -14,4 +14,28 @@ ngrok http http://127.0.0.1:5000 80
...
+
+Features -> Event Subscriptions:
+
+Subscribe to bot events:
+
+- app_mention
+- message.channels
+- message.im
+
+**MAKE SURE TO INVITE THE BOT MANUALLY IN THE SLACK CHANNEL ON THE SLACK APP!**
```
Comment (Squishy123) [inline README.md:41]: Good addition to the readme

File: bot1.py [hunk 2]
```diff
@@ -20,10 +20,11 @@
 client = slack.WebClient(token=os.environ['SLACK_TOKEN'])
 
 # connect the bot to the channel in Slack Channel
-client.chat_postMessage(channel='#bot-dev-chris', text='Send Message Demo')
+client.chat_postMessage(channel='#bot-dev', text='Message from bot (on startup)')
 
 # Get Bot ID
 BOT_ID = client.api_call("auth.test")['user_id']
+print("bot is running as user_id:", BOT_ID)
```
Comment (Squishy123) [inline bot1.py:27]:
```
print("bot is running as user_id:", BOT_ID)
```
could remove these verbose statements for prod

**Comment by JoseArnel** [comment]
:)

**Comment by ghost** [comment]
Woop Woop

**Review by Squishy123 — COMMENTED** [review]
Looks pretty good!

**Pull request merged**